C/wrappedJets/elements/jets.c

```diff
@@ -1,4 +1,4 @@
-#include "primitive/elements/jets.h"
+#include "../../primitive/elements/jets.h"
 #include "../wrappers.h"
 
 WRAP_(version)
```

Comment:
> `COREWRAP_(calculate_issuance_entropy)`

Why are certain Elements jets wrapped without environment? Are those precisely the jets that don't read from the environment?

On the Rust side we always pass the environment to the jet. There is no way of knowing which jets read and which don't read from the environment. If it's not too much work on the Haskell side, let's wrap all Elements jets using `WRAP_`.